src/products/android-l10n/review_Android_strings.md

```diff
@@ -0,0 +1,41 @@
+# Reviewing strings for a new release of Mozilla Android products
+
+Usually, a new release of our Android products means an update to strings. If this is the case, pull requests will be made throughout the release cycle by the mobile team, in order to land these new strings in the [android-l10n](https://github.com/mozilla-l10n/android-l10n) repository in `mozilla-l10n`.
+
+The PRs (also marked as `quarantine`) are reviewed by an l10n-driver - most often by the PM in charge of the project.
+
+Let’s go over some of the steps needed over time in order to review strings correctly for a new release.
+
+## Reviewing the PR
+
+Let's consider Firefox for Android as an example - in fact, these apply to other Android products as well, and they also follow the same train schedule. During the [Android l10n cycle](https://docs.google.com/spreadsheets/d/1hER_w7pOsWSjeBaUqMd_9ClYCCGlwc4jaFYpcBvFE8g/edit#gid=1960046531), the Android team will make pull requests, in order to land new strings for the upcoming release.
+
+Let’s consider a past PR [here](https://github.com/mozilla-l10n/android-l10n/pull/391/files).
+
+The first file present is `_meta/mozilla-mobile-fenix.json`. This file is part of a [changeset](https://github.com/mozilla-mobile/fenix/commit/76c0c4ad1425cfd79c25d707921e8185620ad080) from the Fenix repository, and is used to generate the strings in export PRs. Updates to the `main` section are expected and can be ignored.
```

Comment:
This is still not really accurate for someone who's never seen this project (and is the target of these docs).

```suggestion
The first file present is `_meta/mozilla-mobile-fenix.json`. This file is managed by the automation syncing content between code and l10n repositories, and it’s used to track which [changeset](https://github.com/mozilla-mobile/fenix/commit/76c0c4ad1425cfd79c25d707921e8185620ad080) from the `fenix` repository was used to export the strings in this pull requests. `main` is the name of the branch from which strings are extracted, `fenix` is the internal project name of Firefox for Android. Updates to the value of `main` can be ignored, while changes to the name of the branch need to be coordinated with developers and release engineering.
```

Reply:
Oh ok will update, didn't realize we would need to get into these details